packages/airnode-examples/src/cli.ts

```diff
@@ -32,7 +33,7 @@ export const runShellCommand = (command: string) => {
 };
 
 export const cliPrint = {
-  info: (text: string) => console.log(chalk.bold.white(text)),
-  warning: (text: string) => console.log(chalk.bold.hex('#FFA500')(text)), // Orange color
-  error: (text: string) => console.log(chalk.bold.red(text)),
+  info: (text: string) => log(chalk.bold.white(text)),
```

Comment:
I've been thinking about extracting other common utilities from Airnode and was thinking that it would be nice to extract the entire logger from the node package. So this would include the timestamp, log level, message and any metadata.

I was also thinking that a nice API would look like:

```ts
export const logger = {
  log: (...) => { ... },
  debug: (...) => { ... },
  info: (...) => { ... },
  warn: (...) => { ... },
  error: (...) => { ... }
};

import { logger } from '@api3/airnode-utilities';

logger.info('Hello world');
```

To me this is quite a nice intuitive, "traditional" logging API. What do you think? cc @Siegrift

Reply:
The logger would also be able to format as either JSON or regular text like the current node logger does

Reply:
I'd rather use an existing solution like winston or pino. The logging is actually pretty hard concept (e.g. performance) to get right and I'd rather use some battle tested solution.

Reply:
> I've been thinking about extracting other common utilities from Airnode and was thinking that it would be nice to extract the entire logger from the node package. So this would include the timestamp, log level, message and any metadata.

I should have given more context to implementing this as a separate package, but my idea was exactly this, to move all the logging over to utilities eventually. Airkeeper has been using these facilities.

Reply:
> I'd rather use an existing solution like winston or pino.

pino looks nice. My only concern is another 3rd-party dependency and then a supply chain attack like Log4j.